README.md

```diff
@@ -67,7 +67,7 @@ Basic Usage
 
 ## [Shallow Rendering](/docs/api/shallow.md)
 
-```jsx
+```javascript
```

Comment:
Because right now syntax highlighting is broken with `jsx` instead of `javascript`.

That's how it looked before fix:
![](http://cl.ly/3u422h1g161p/Image%202016-06-10%20at%202.36.04%20AM.png)

and after:
![](http://cl.ly/0v2F382F1d02/Image%202016-06-10%20at%202.35.02%20AM.png)